TACC/protx-dashboard
# task/COOKS-343-Reconfig-Red-Line-on-County-Histogram-When-Less-Than-0

## Overview: ##
Updated analytics plot to show 0 if pred_per_100 < 0

## Related Jira tickets: ##

* [COOKS-343](https://jira.tacc.utexas.edu/browse/COOKS-123)

## Summary of Changes: ##
Added max function so that if v_line is less than zero, return 0

## Testing Steps: ##
1. Open analytics.db to compare with plot
2. Use [this resource](https://www.dshs.texas.gov/chs/info/info_txco.shtm) to find FIPS code for each county to read the DB
3. Go to [https://cep.test/protx/dash/analytics](https://cep.test/protx/dash/analytics) and click around to find counties with pred_per_100k less than 0. Some example counties are:

- Wheeler - GeoId 48483
- Dawson - GeoId 48115
- Briscoe - GeoId 48045
- Martin - GeoId 48317

4. Make sure the red line is at 0 rather than not displayed at all. 

## UI Photos
Martin County has a pred_per_100k in the DB listed as 0.0006237133894844

<img width="1757" alt="Screen Shot 2022-11-21 at 9 07 03 AM" src="https://user-images.githubusercontent.com/96220951/203088805-38fcfc15-4308-49d5-b38d-2fe4f802ad8c.png">


Maverick county has a pred_per_100k in the DB listed as -89.5924433487155. It now shows as 0 on the histogram.
<img width="1509" alt="Screen Shot 2022-11-21 at 9 09 38 AM" src="https://user-images.githubusercontent.com/96220951/203089349-1a56d158-f041-4efe-8fc9-8d538d6f5e85.png">


## Notes: ##
